### PR TITLE
Add Guide for changing the Default Shell

### PR DESCRIPTION
**Pull request by MonsterObserver**

### CODE REVIEW

File: _posts/2023-01-19-change-default-shell.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+## Revert your Default Shell to Bash
+
+- You can run the following command to revert your Default Shell back to Bash:
+
+```bash
+chsh -s /usr/bin/bash
+```
+
+- You will now have to reboot to apply the changes. Simply run `reboot`.
```
Comment (MonsterObserver) [inline _posts/2023-01-19-change-default-shell.md:65]:
```suggestion
- You will now have to reboot the system to apply the changes. Simply run `reboot`.
```

**Review by wolfiediscord — COMMENTED** [review]
"Firstly" sounds a little weird. "First" sounds better and fits with the handbook better.

File: _posts/2023-01-19-change-default-shell.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+## Introduction
+
+Vanilla OS allows you to change your Default Shell to better fit your workflow. This guide will show you how to change your Default Shell to [**`Zsh`**](https://zsh.sourceforge.io/) or [**`Fish`**](https://fishshell.com/).
+
+**_Note_**:- Changing the Default Shell is only recommended for advanced users.
+
+## Change your Default Shell to Zsh
+
+- Firstly, we need to install Zsh on the Host System with the following command:-
```
Comment (MonsterObserver) [inline _posts/2023-01-19-change-default-shell.md:19]:
```suggestion
- First, we need to install Zsh on the Host System with the following command:-
```

File: _posts/2023-01-19-change-default-shell.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+chsh -s /usr/bin/zsh
+```
+
+- After changing the Default Shell to Zsh, we need to reboot the system one last time to apply the changes. Simply run `reboot` again.
+
+- The next time you open your Terminal Emulator you will be prompted to setup Zsh.
+
+## Change your Default Shell to Fish
+
+- Firstly, we need to install Fish on the Host System with the following command:-
```
Comment (MonsterObserver) [inline _posts/2023-01-19-change-default-shell.md:39]:
```suggestion
- First, we need to install Fish on the Host System with the following command:-
```

**Review by MonsterObserver — COMMENTED** [review]
Changing Firstly to First

**Review by mirkobrombin — APPROVED** [review]

**Pull request merged**